vitest.shared.mjs

```diff
@@ -1,9 +1,12 @@
 import inspector from 'node:inspector';
+import process from 'node:process';
 import { defineConfig } from 'vitest/config';
 import pkg from './package.json';
-
 export default defineConfig({
     test: {
+        env: {
```

Comment:
```suggestion
        // We can't set `NODE_ENV` directly (i.e. `NODE_ENV=production yarn test`), because some packages
        // set their env to jsdom, and NODE_ENV=production + jsdom causes `node:` imports to break
        env: {
```